### DISCUSSION TITLE
Tables with checkboxes

### DISCUSSION THREAD

**Discussion by vankhoawin** [twilio-labs/paste#1177] [Q&A] [answered]
Apologies if this was previously asked, I tried doing some initial searching.

Phone Numbers is migrating this modal currently in the SIP Trunking page to Paste. My question is, will Paste support checkboxes in tables (with a select-all checkbox)? Or should this be something built using the individual components?

![image](https://user-images.githubusercontent.com/6936736/107301348-2d8b2400-6a30-11eb-941b-2880438a2738.png)

**Comment by kate-porter** [comment, marked as answer]
Hey @vnguyen94 - this functionality won't be built into Table (selectable rows will come when we build Data Grid), but you should be able to compose this yourself with Table and Checkboxes. The folks in IoT are doing something similar with their Bulk Actions work, so I'd recommend connecting with them.

@ishah-twilio - would you be able to share your Bulk Actions designs with Van and connect him with the engineers who are building it out so y'all can share similar patterns?

**Reply by vankhoawin** [reply]
gotcha, sounds good. and also could you link (or DM) me the Data Grid ticket so we can track it for migration once it's ready?

**Reply by kate-porter** [reply]
Just DM'd you a link to the epic!

**Reply by ishah-twilio** [reply]
Just DM'd you details about IoT Bulk action designs. Let me know if you have any questions.

**Reply by vankhoawin** [reply]
thank you all, this was extremely helpful!

**Reply by kate-porter** [reply]
Thank you, @ishah-twilio! And happy to help, @vnguyen94!
